app/src/main/java/com/nextcloud/talk/controllers/ChatController.kt

```diff
@@ -1432,15 +1459,82 @@ class ChatController(args: Bundle) : BaseController(args), MessagesListAdapter
                         }
                         true
                     }
+                    R.id.action_delete_message -> {
+                        ncApi?.deleteChatMessage(
+                                credentials,
+                                ApiUtils.getUrlForMessageDeletion(conversationUser?.baseUrl, roomToken, message?.id)
+                        )?.subscribeOn(Schedulers.io())
+                                ?.observeOn(AndroidSchedulers.mainThread())
+                                ?.subscribe(object : Observer<ChatOverallSingleMessage> {
+                                    override fun onSubscribe(d: Disposable) {
+                                    }
+
+                                    override fun onNext(t: ChatOverallSingleMessage) {
+                                        if (t.ocs.meta.statusCode == 202) {
+                                            Toast.makeText(context, R.string.nc_delete_message_leaked_to_matterbridge,
+                                                    Toast.LENGTH_LONG).show()
+                                        }
+                                    }
+
+                                    override fun onError(e: Throwable) {
+                                        Log.e(TAG, "Something went wrong when trying to delete message with id " +
```

Comment:
Do you want to show a Toast in this case as well?

Reply:
check